Tests/RosalindTests/__Snapshots__/RosalindAcceptanceTests/apple_ios_app.1.json

```diff
@@ -0,0 +1,53 @@
+{
+  "app" : {
+    "children" : [
+      {
+        "unknown" : {
+          "children" : [
+
+          ],
+          "path" : "App",
+          "size" : 70880
```

Comment:
I need to adjust the matcher to ignore this piece, since I don't expect the size to be deterministic across runs.

Reply:
And the same goes for the shasum :)

Reply:
I wonder if it would be, after all, better to include the binaries in the repo. I'd expect them to be quite small and we can zip them if needed. The acceptance tests would run faster and the snapshots would be more stable.

Reply:
The stability shouldn't be a problem if we pick the right testing approach. We should have a test that ensures the schema is the right one, and have a unit test that ensures we calculate the shasum as we should. With those in place, I lean on doing everything e2e, and iterate on it if we notice the experience is degraded due to performance or other reasons.

Reply:
For one, we can't run the acceptance tests on Linux. I don't really see a strong benefit of compiling the projects as part of the acceptance tests instead of running them against binary, but if you feel strongly, we can keep it as-is.